web/test/pages/onboarding/onboarding-owning.test.tsx

```diff
@@ -190,6 +193,43 @@ describe("onboarding - owning a business", () => {
     });
   });
 
+  it("prevents user from moving after Step 4 if you have not entered number of employees", async () => {
+    const initialUserData = createEmptyUserData(generateUser({}));
+    const newark = generateMunicipality({ displayName: "Newark" });
+    const { subject, page } = renderPage({
+      userData: initialUserData,
+      municipalities: [newark],
+    });
+
+    page.chooseRadio("has-existing-business-true");
+    await page.visitStep2();
+    await page.visitStep3();
+    page.fillText("Business name", "A business");
+
+    await page.visitStep4();
+    expect(subject.getByTestId("step-4")).toBeInTheDocument();
+    page.selectByText("Location", "Newark");
+    page.clickNext();
+
+    await waitFor(() => {
+      subject.getByText(templateEval(OnboardingDefaults.errorTextMinimumNumericField, { length: "7" }));
+      expect(subject.queryByTestId("toast-alert-ERROR")).toBeInTheDocument();
+    });
+
+    page.fillText("Existing employees", "123");
+    page.clickNext();
+    await waitFor(() => {
+      expect(mockRouter.mockPush).not.toHaveBeenCalledWith("/roadmap");
+      expect(subject.getByTestId("step-4")).toBeInTheDocument();
+      subject.getByText(templateEval(OnboardingDefaults.errorTextMinimumNumericField, { length: "7" }));
```

Comment:
Not sure I understand this. Why is 123 not a valid number pf existing employees?